Packs/CommonScripts/Scripts/CheckIfSubdomain/CheckIfSubdomain.py

```diff
@@ -0,0 +1,39 @@
+import tldextract
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def main():
```

Comment:
Please also add here `try` and `except` that uses `return_error`, for edge cases.

Reply:
Done